pipelines/controller/src/main/java/org/openmrs/analytics/HiveTableManager.java

```diff
@@ -76,26 +75,75 @@ public void createResourceTables(
   private void createResourceTable(
```

Comment:
Naming suggestion: there are too many `createResourceTable[s]`, let's rename this one to `createResourceAndCanonicalTable` or something like that to reflect the extra piece that this does on top of the next `createResourceTable` function.